src/test/resources/ReadMem.hex

```diff
@@ -0,0 +1,3 @@
+// comment
+ab
+CD eF // any whitespace should be treated as a separator
```

Comment:
Did you mean for this to not have final newline as part of the test?

Reply:
Yes, this is intentional